src/components/ConfigSelect/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+
+import { 
+  ConfigLabelText,
+  ConfigLableContainer,
+  ConfigMetricWrapper,
+  SwitchContainer,
+  Picker
+} from '../Globals/ConfigMetric';
+
+interface ConfigSelectProps {
+  label: string;
+  unit: string | undefined;
+  pickerItems: Array<{
+    value: string | number;
+  }>
+  isDisabled: boolean;
+  settingsHandler: (key: string, value: string | boolean) => void;
+  selectedValue?: string;
+}
+
+const ConfigSelect = ({
+  label,
+  unit,
+  pickerItems,
+  isDisabled,
+  settingsHandler,
+  selectedValue
+}: ConfigSelectProps ) => {
+  return (
+    <ConfigMetricWrapper>
+    <ConfigLableContainer>
```

Comment:
could you check the indentation of this file? `ConfigLabelContainer` should be indented more than it's parent `ConfigMetricWrapper`